EncoreTickets.SDK.Tests/UnitTests/Api/Results/ApiResultTests.cs

```diff
@@ -55,6 +55,42 @@ public void ConstructorWith5Args_IfUnsuccessfulResponse_InitializesCommonPropert
             Assert.AreEqual(thrownException, result.ApiException);
         }
 
+        [Test]
+        public void ConstructorWithMultipleErrors_IfSuccessfulResponse_InitializesCommonProperties()
+        {
+            var data = new[] { new object(), new object(), };
+            var response = RestResponseFactory.GetSuccessResponse();
+            var context = It.IsAny<ApiContext>();
+            var errors = (IEnumerable<Error>)null;
+
+            var result = new ApiResult<object>(data, response, context, errors);
+
+            Assert.AreEqual(context, result.Context);
+            Assert.AreEqual(response, result.RestResponse);
+            Assert.AreEqual(true, result.IsSuccessful);
+            Assert.AreEqual(default, result.ApiException);
+            Assert.AreEqual(data, result.DataOrException);
+            Assert.AreEqual(data, result.DataOrDefault);
+        }
+
+        [Test]
+        public void ConstructorWithMultipleErrors_IfUnsuccessfulResponse_InitializesCommonProperties()
+        {
```

Comment:
The test is named as unsuccessful but you create a successful response inside it. And even if the aim of this test is checking constructor behavior when a response is successful but some errors are arguments so you need to compare result.Context with not any value